internal/grpcapi/order_service.go

```diff
@@ -0,0 +1,34 @@
+package grpcapi
+
+import (
+	"context"
+
+	pb2 "github.com/Sugar-pack/orders-manager/pkg/pb"
+	"github.com/Sugar-pack/users-manager/pkg/logging"
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+)
+
+type OrderService struct {
+	pb2.OrdersManagerServiceServer
+	dbConn *sqlx.DB
+}
+
+func (s OrderService) ReceiveOrder(cxt context.Context, order *pb2.Order) (*pb2.OrderTnxResponse, error) {
```

Comment:
Receive means "to get", but here we INSERT order. Suggest to rename method.

Reply:
Done